js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -30,6 +31,42 @@ import {
  * @property {boolean} isReady Whether the campaign data and the billing setting are ready for completing the paid ads setup.
  */
 
+const defaultPaidAds = {
+	amount: 0,
+	countryCodes: [],
+	isValid: false,
+	isReady: false,
+};
+
+/**
+ * Resolve the initial paid ads data from the given paid ads data with the loaded target audience.
```

Comment:
:nail_care: WDYT of
```suggestion
 * Resolve the updated state of paid ads data from the previous state and the loaded target audience.
```
is it accurate? sound better or worse?

Reply:
Please refer to https://github.com/woocommerce/google-listings-and-ads/pull/1683#discussion_r979548972.